packages/radix-vue/src/Tooltip/TooltipContentImpl.vue

```diff
@@ -77,30 +77,20 @@ const ariaLabel = computed(() => {
   return content
 })
 
-const popperContentProps = computed(() => ({
-  as: props.as,
-  asChild: props.asChild,
-  side: props.side,
-  sideOffset: props.sideOffset,
-  align: props.align,
-  alignOffset: props.alignOffset,
-  avoidCollisions: props.avoidCollisions,
-  collisionBoundary: props.collisionBoundary,
-  collisionPadding: props.collisionPadding,
-  arrowPadding: props.arrowPadding,
-  sticky: props.sticky,
-  hideWhenDetached: props.hideWhenDetached,
-}))
+const popperContentProps = computed(() => {
+  const { ariaLabel: _, ...restProps } = props
+  return restProps
+})
```

Comment:
I have also corrected this computed. The effect should be the same, but with less code